src/content/reference/hi/p5/mag.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: mag
+module: गणित
+submodule: गणना
+file: src/math/calculation.js
+description: >
+  <p>एक वेक्टर के परिमाण, या लंबाई की गणना करता है।</p>
+
+  <p>एक वेक्टर के बारे में विभिन्न तरीकों से सोचा जा सकता है। एक दृष्टिकोण में, एक वेक्टर अंतरिक्ष में एक बिंदु है। वेक्टर के घटक, 
```

Comment:
Is Antriksh correct usage here, do check once

Reply:
changed with jagha